webapp/client/src/components/FormFieldDropDown.js

```diff
@@ -0,0 +1,97 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import FormFieldScaffolding from "./FormFieldScaffolding";
+import FieldLabel from "./FieldLabel";
+import selectIcon from "../assets/icons/select_icon.svg";
+
+const DropDown = styled.div`
+  & .steuerlotse-select {
+    border: 2px solid var(--border-color);
+    border-radius: 0;
+    background-image: url(${selectIcon});
+    background-repeat: no-repeat;
+    background-size: 0.75rem;
+    min-height: 55px;
+  }
+
+  & .steuerlotse-select:focus {
+    border: 2px solid var(--focus-border-color);
+    box-shadow: 0 0 0 2px var(--focus-color);
+  }
+`;
+
+function FormFieldDropDown({
+  fieldName,
+  fieldId,
+  options,
+  defaultOption,
+  selectedValue,
+  required,
+  label,
+  details,
+  errors,
+  onChangeHandler,
+}) {
+  return (
+    <FormFieldScaffolding
+      {...{
+        fieldName,
+        errors,
+        cols: "6",
+      }}
+      hideLabel
+      render={() => (
+        <DropDown>
+          <FieldLabel {...{ label, fieldId, details }} />
+          <select
+            id={fieldId}
+            className="custom-select steuerlotse-select"
+            name={fieldId}
+            defaultValue={selectedValue}
+            required={required}
+            autoFocus={Boolean(errors.length)}
```

Comment:
I think thos should also have a `autofocus` parameter that gets tested here? Or is it a concious decision to leave the `autofocus` out here because we want to get rid of it in the future anyway?

Reply:
That was basically my thinking